### ISSUE TITLE
installation problem

### ISSUE THREAD

**Issue by singhjaspreetb** [trustedsec/social-engineer-toolkit#728]
it shows 
:~/setoolkit$ sudo pip3 install -r requirements.txt
Collecting pexpect
  Using cached pexpect-4.8.0-py2.py3-none-any.whl (59 kB)
Collecting pycrypto
  Using cached pycrypto-2.6.1.tar.gz (446 kB)
Collecting requests
  Using cached requests-2.23.0-py2.py3-none-any.whl (58 kB)
Collecting pyopenssl
  Using cached pyOpenSSL-19.1.0-py2.py3-none-any.whl (53 kB)
Collecting pefile
  Using cached pefile-2019.4.18.tar.gz (62 kB)
Collecting impacket
  Using cached impacket-0.9.21.tar.gz (1.3 MB)
Collecting qrcode
  Using cached qrcode-6.1-py2.py3-none-any.whl (31 kB)
Collecting pillow
  Using cached Pillow-7.1.2-cp38-cp38-manylinux1_x86_64.whl (2.1 MB)
Collecting pymssql<3.0
  Using cached pymssql-2.1.4.tar.gz (691 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-7dtd7bko/pymssql/setup.py'"'"'; __file__='"'"'/tmp/pip-install-7dtd7bko/pymssql/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-7dtd7bko/pymssql/pip-egg-info
         cwd: /tmp/pip-install-7dtd7bko/pymssql/
    Complete output (7 lines):
    /usr/lib/python3/dist-packages/setuptools/dist.py:46: DistDeprecationWarning: Do not call this function
      warnings.warn("Do not call this function", DistDeprecationWarning)
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-7dtd7bko/pymssql/setup.py", line 88, in <module>
        from Cython.Distutils import build_ext as _build_ext
    ModuleNotFoundError: No module named 'Cython'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

help me  to install it

**Comment by sharad1126** [comment]
try `pip3 install cython` and then use the `pip3 install -r requirements.txt`. Also won't suggest you using sudo while doing pip3.

**Comment by ErickGiffoni** [comment]
I'm having the same installation problem about Cython :/

**Comment by ErickGiffoni** [comment]
> try `pip3 install cython` and then use the `pip3 install -r requirements.txt`. Also won't suggest you using sudo while doing pip3.

That worked for me to install Cython.

Then I ran `pip3 install -r requirements.txt` and I got this error below : 

```
gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/Users/erick.giffoni/anaconda3/include -arch x86_64 -I/Users/erick.giffoni/anaconda3/include -arch x86_64 -I/usr/local/include -I/Users/erick.giffoni/anaconda3/include/python3.6m -c src/_mssql.c -o build/temp.macosx-10.7-x86_64-3.6/src/_mssql.o -DMSDBLIB
    src/_mssql.c:607:10: fatal error: 'sqlfront.h' file not found
    #include "sqlfront.h"
             ^~~~~~~~~~~~
    1 error generated.
    error: command 'gcc' failed with exit status 1
```
@osm1n

**Comment by MittKevv** [comment]
> try `pip3 install cython` and then use the `pip3 install -r requirements.txt`. Also won't suggest you using sudo while doing pip3.

it helped in my case

**Comment by Sourabh3131** [comment]
Same problem persisting┐( ∵ )┌

**Comment by Valinwolf** [comment]
I was having this issue installing SEToolkit on Win10 x64 and after much research it seemed to be an issue with the Python version I was using, 3.9. After rolling back to 3.7, installed via the VS installer, pymssql seemed to install without the need to build anything, however I [prepped my terminal](https://github.com/pymssql/pymssql/issues/372#issuecomment-397184839), so I don't know if that had anything to do with it. After that, I seemed to have some issues with pycrypto. I solved that by using the VS installer to install VC++ 2017 and [the VC tools command prompt like a SO user suggested](https://stackoverflow.com/a/41843310).

TL;DR: Downgrade Python version to 3.7 or 3.6 and install VC++ 2017 and [install pycrypto using this post](https://stackoverflow.com/a/41843310). If pymssql doesn't install try [this post](https://github.com/pymssql/pymssql/issues/372#issuecomment-397184839).

_P.S.: Perhaps it should be in the readme that SEToolkit needs a specific version, at least on Window$?_
